pinot-core/src/main/java/com/linkedin/pinot/core/operator/aggregation/function/AggregationFunctionFactory.java

```diff
@@ -91,7 +92,7 @@ public static AggregationFunction getAggregationFunction(String functionName, Se
         return new DistinctCountHLLAggregationFunction();
 
       case FASTHLL_AGGREGATION_FUNCTION:
-        return new FastHllAggregationFunction(segmentMetadata.getHllLog2m());
+        return new FastHllAggregationFunction(HllConstants.DEFAULT_LOG2M);
```

Comment:
Please add a TODO here to fix FastHll function. And handle HyperLogLog object merge if log2m values are different.